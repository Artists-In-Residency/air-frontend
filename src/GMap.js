import React, { Component } from 'react'
import { WrappedMap } from './GMapRender.js';

export default class GMap extends Component {

  render() {
    return (
      <div style={{ width: '100vw', height: '70vh' }} >
        <WrappedMap 
          googleMapURL={`https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places&key=${
          process.env.REACT_APP_GOOGLE_MAPS_API}`}
          loadingElement={<div style={{ height: "100%" }} />}
          containerElement={<div style={{ height: "100%" }} />}
          mapElement={<div style={{ height: "100%" }} />}
        />
      </div>
    )
  }
}