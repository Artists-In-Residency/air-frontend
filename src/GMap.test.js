import React from 'react';
import GMap from './GMap.js';
import renderer from 'react-test-renderer';

it('header renders correctly', () => {
  const tree = renderer
    .create(<GMap/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});