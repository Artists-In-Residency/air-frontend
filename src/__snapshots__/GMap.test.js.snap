// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div
  style={
    Object {
      "height": "70vh",
      "width": "99%",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
</div>
`;
